performance_test/src/experiment_execution/analyze_runner.cpp

```diff
@@ -54,6 +75,59 @@ AnalyzeRunner::AnalyzeRunner()
     m_sub_runners.push_back(DataRunnerFactory::get(m_ec.topic_name(), m_ec.com_mean(),
       RunType::SUBSCRIBER));
   }
+
+#ifdef ODB_FOR_SQL_ENABLED
+#ifdef PERFORMANCE_TEST_ODB_SQLITE
+  m_db = std::unique_ptr<odb::core::database>(new odb::sqlite::database(
+        m_ec.db_name(), SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE));
+  {
+    odb::core::connection_ptr c(m_db->connection());
+    c->execute("PRAGMA foreign_keys=OFF");
+    odb::core::transaction t(c->begin());
+    try {
+      m_db->query<ExperimentConfiguration>(false);
+    } catch (const odb::exception & e) {
```

Comment:
This code block can be made into a function since its being repeated for each database?

Reply:
ok, I didn't move it to into function but I squeezed it as much as possible to not repeat the code.

Reply:
Okay for PERFORMANCE_TEST_ODB_MYSQL and PERFORMANCE_TEST_ODB_PGSQL I guess you can just write the `m_db` initialization just once ?

Reply:
You have `new odb::mysql::database` and `new odb::pgsql::database` so it can't be written just once. I could move the arguments cause they are the same for both, but I'd have to add other option for mysql or pgsql. I don't think it will be better... I changed #ifdefs to contain #elifs so it's even less code now.

Reply:
Oh I got it now. Sorry I did not notice that before. Looks good then